boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <App_SharedExitCode.h>
+
+struct Imu;
+
+/**
+ * Allocate and initialize an IMU.
+ * @param get_exit_code A function that can be called to get an exit code
+ * corresponding     to the last reading of the data from the Imu
+ * @param get_acceleration_x A function that can be called to get the
+ * acceleration (m/s^2) on the x-axis.
+ * @param get_acceleration_y A function that can be called to get the
+ * acceleration (m/s^2) on the y-axis.
+ * @param get_acceleration_z A function that can be called to get the
+ * acceleration (m/s^2) on the z-axis.
+ * @param min_acceleration The min acceleration the Imu should read for a given
+ * axis, in m/s^2
+ * @param max_acceleration The max acceleration the Imu should read for a given
+ * axis, in m/s^2
+ * @return The created IMU whose ownership is given to the caller.
+ */
+struct Imu *App_Imu_Create(
+    float (*get_acceleration_x)(void),
+    float (*get_acceleration_y)(void),
+    float (*get_acceleration_z)(void),
+    float min_acceleration,
+    float max_acceleration);
+
+/**
+ * Deallocate the memory used by the given Imu
+ * @param imu The Imu to deallocate
+ */
+void App_Imu_Destroy(struct Imu *imu);
+
+/**
+ * Get acceleration (m/s^2) measured on the x-axis from the given Imu.
+ * @param imu The Imu to get the x-axis acceleration from.
+ * @return The acceleration (m/s^2) measured on the x-axis.
+ */
+float App_Imu_GetAccelerationX(const struct Imu *imu);
+
+/**
+ * Get acceleration (m/s^2) measured on the y-axis from the given Imu.
+ * @param imu The Imu to get the y-axis acceleration from.
+ * @return The acceleration (m/s^2) measured on the y-axis.
+ */
+float App_Imu_GetAccelerationY(const struct Imu *imu);
+
+/**
+ * Get acceleration (m/s^2) measured on the z-axis from the given Imu.
+ * @param The Imu to get the z-axis acceleration from.
+ * @return The acceleration (m/s^2) measured on the z-axis.
+ */
+float App_Imu_GetAccelerationZ(const struct Imu *imu);
+
+/**
+ * Get InRangeCheck corresponding to acceleration on the x-axis for the given
+ * Imu
```

Comment:
```suggestion
 * Get the acceleration_x in-range check from the given imu
```

This applies to the javadocs below as well

Reply:
ok done